pkg/trace/agent/run.go

```diff
@@ -30,14 +24,13 @@ const messageAgentDisabled = `trace-agent not enabled. Set the environment varia
 DD_APM_ENABLED=true or add "apm_config.enabled: true" entry
 to your datadog.yaml. Exiting...`
 
+// ConfigPath specifies the path to the configuration file. It is set via the
+// command's -c / --cfgpath persistent flag.
```

Comment:
Is this a breaking change from our `-config` right?

Reply:
Yes.